src/api/app/views/webui/request/_superseded_by_message.html.haml

```diff
@@ -3,28 +3,26 @@
     .ui-state-info.ui-corner-all.ui-widget-shadow.padding-10pt#request-supersedes-hint
       %span.ui-icon.ui-icon-info
         This request supersedes:
-        - superseding.each do |bs_request|
+        - superseding.each_with_index do |bs_request, index|
           = link_to "request #{bs_request.number}", request_show_path(number: bs_request.number)
-          (
-          = link_to 'Show diff', request_show_path(number: bs_request.superseded_by, diff_to_superseded: bs_request.number)
-          )
+          = surround '(', ')' do
+            = link_to 'Show diff', request_show_path(number: bs_request.superseded_by, diff_to_superseded: bs_request.number)
+          = ', ' if index < superseding.length - 1
 - if superseded_by.present?
   .grid_16.alpha.omega.box-invisible
     .ui-state-info.ui-corner-all.ui-widget-shadow.padding-10pt#request-superseded-by-hint
       %span.ui-icon.ui-icon-info
         This request is superseded by
         = link_to "request #{superseded_by}", request_show_path(number: superseded_by)
-        (
-        = link_to 'Show diff', request_show_path(number: superseded_by, diff_to_superseded: number)
-        )
-- if diff_to_superseded.present?
+        = surround '(', ')' do
+          = link_to 'Show diff', request_show_path(number: superseded_by, diff_to_superseded: number)
+- if diff_to_superseded
```

Comment:
this should be fix in the first commit and not in an extra one, that is where you introduce it :wink: